man/systemd.network.xml

```diff
@@ -899,6 +895,47 @@
     </refsect1>
 
   <refsect1>
+    <title>[IPv6AcceptRouterAdvertisements] Section Options</title>
```

Comment:
well, the RA client side option in [Network] is called IPv6AcceptRouterAdvertisements= currently, and I'd like to keep the section name in sync with that.

We might eventually add an RA sender, too, in order to implement prefix delegation properly. if we do that [IPv6RouterAdvertisement] would become unclear in its meaning: that section could be about RA reception as well as about RA submission... It's kinda like [DHCP] And [DHCPServer] in that regard.

I also think the section name is a bit long, but my suggestion would be to maybe rename both the option and the section then, to IPv6AcceptRA= and [IPv6AcceptRA]. Of course we'd have to keep IPv6AcceptRouterAdvertisements= as a compat alias around then.